core/src/test/java/io/ultrabrew/metrics/data/BasicGaugeAggregatorTest.java

```diff
@@ -112,4 +113,12 @@ public void testGrowTableWithMaxCapacity() {
     assertEquals(3, aggregator.size());
     assertEquals(3, aggregator.capacity()); // caped at the max capacity.
   }
+
+  @Test
+  void testDefaultCardinality() {
+    BasicGaugeAggregator aggregator = new BasicGaugeAggregator("test");
+    int maxCapacity = Deencapsulation.getField(aggregator, "maxCapacity");
```

Comment:
Remove Deencapsulation usage

Reply:
We will address them as part of https://github.com/ultrabrew/metrics/issues/12